### DISCUSSION TITLE
Specs no longer visible in 10.5.0

### DISCUSSION THREAD

**Discussion by BryanShekletonwv** [cypress-io/cypress#23460] [Questions and Help]
Hi, Since upgrading from cypress 10.4.0 to 10.5.0 the specs are no longer visible: 
![image](https://user-images.githubusercontent.com/97227273/185602672-667e741b-3754-48bf-9c71-c5d0bf776a98.png)

Inside settings I can see 49 matches:
![image](https://user-images.githubusercontent.com/97227273/185602815-46e36697-2f60-47db-8706-c0a33e27c31a.png)

Has anyone else seen this?

**Comment by lmiller1990** [comment]
Bizarre. Cypress 10.5 is working fine for me. 

Can you open the devtools? Any unexpected errors/warnings?

Could you please share your `cypress.config`? Or, a minimal repro? I tried reproducing (see #23486) but no luck.

PS: 10.6.0 is out - you could try that (although not aware of any bugs around the spec list).

**Comment by lmiller1990** [comment]
Issue here, too: https://github.com/cypress-io/cypress/issues/23486

**Comment by marktnoonan** [comment]
Hi @BryanShekletonwv it looks like one cause of this state is having a `component` key in your Cypress config file, but not having a `devServer` object defined inside. Some code was added in 10.5.0 that can error in this situation (more details in https://github.com/cypress-io/cypress/issues/23486). 

We expect to fix that in the next release but please confirm if that is the cause for you. To check, just remove the `component` key from your config file. Thanks!
